app/controllers/concerns/api_actionable.rb

```diff
@@ -58,8 +57,8 @@ def handle_custom_actions
 
           eval("def custom_api_action.run_custom_action(api_action: , api_namespace:, api_resource: , current_visit: , current_user: nil); #{custom_action.method_definition};end;")
           response = custom_api_action.run_custom_action(api_action: custom_action, api_namespace: @api_namespace, api_resource: @api_resource, current_visit: current_visit, current_user: current_user)
-          response = response.present? ? response.as_json : "everything went well."
-          custom_action.update(lifecycle_stage: 'complete', lifecycle_message: response)
+
+          custom_action.update(lifecycle_stage: 'complete', lifecycle_message: response.to_json)
```

Comment:
is there a reason for doing `response.to_json` ? 

What if the implicit return value was a `boolean` or `string` ? 

A better approach would just be to do `response.to_s` or just `response`-- any opinions on this @Pralish ?

Reply:
@donrestarone, converting boolean or string or even nil to json won't raise any errors. I prefer to_json because we can parse it easily if you ever wish to use the response.

But if we wish to just see the response, to_s looks cleaner.

Reply:
@alis-khadka I stand corrected! 

Thanks for the clarification @Pralish -- this doesn't need to be changed in that case